cmake_templates/AndroidManifest.xml.in

```diff
@@ -65,40 +61,36 @@
             <meta-data 
                 android:name="android.app.arguments" 
                 android:value="-- %%INSERT_APP_ARGUMENTS%% --" />
-
-            <!-- Background running -->
-            <!-- Warning: changing this value to true may cause unexpected crashes if the
-                          application still try to draw after
-                          "applicationStateChanged(Qt::ApplicationSuspended)"
-                          signal is sent! -->
-            <meta-data android:name="android.app.background_running" android:value="false"/>
```

Comment:
are you confident we can remove this one?

Reply:
It was a copy/paste from Qt's template, they removed it back in 2021 https://github.com/qt/qtbase/commit/a1e7c441de16cb4e8d6bdac39160759116ecd233 - we will surely test this properly